src/Gui/Application.cpp

```diff
@@ -821,27 +821,27 @@ void Application::slotRelabelDocument(const App::Document& Doc)
 
 void Application::slotRenameDocument(const App::Document& Doc)
 {
-    std::map<const App::Document*, Gui::Document*>::iterator doc = d->documents.find(&Doc);
+    auto doc = d->documents.find(&Doc);
 #ifdef FC_DEBUG
-    assert(doc!=d->documents.end());
+    assert(doc != d->documents.end());
 #endif
 
     signalRenameDocument(*doc->second);
 }
 
 void Application::slotShowHidden(const App::Document& Doc)
 {
-    std::map<const App::Document*, Gui::Document*>::iterator doc = d->documents.find(&Doc);
+    auto doc = d->documents.find(&Doc);
```

Comment:
I think this type is too opaque, you have to know too much about `d->documents` -- I'd switch this one back.